### PR TITLE
[FIX] Fixed a small bug in freesurfer label2annot fill_thresh specs

### PR DESCRIPTION
**Pull request by achetverikov**
Fixes a small bug in freesurfer label2annot fill_thresh specs (should be %f instead of %.f)

### CODE REVIEW

**Comment by effigies** [comment]
Please merge master, run `make specs`, and commit the updated auto tests.

**Comment by effigies** [comment]
Hi @achetverikov do you have a second to give this a shot? Just to be explicit about what I was suggesting:

1) Make sure you have a remote pointing to the main repository (often called `upstream`):

```Shell
git remote add upstream https://github.com/nipy/nipype.git
```

2) Fetch `upstream` and merge `upstream/master`

```Shell
git fetch upstream
git merge upstream/master
```

3) Generate auto tests, add and push

```Shell
make specs
git add nipype/interfaces/freesurfer/tests/*
git commit -m 'TEST: make specs'
git push
```

**Comment by achetverikov** [comment]
OK, done. Thanks for the detailed instructions, @effigies .

File: nipype/interfaces/freesurfer/tests/test_auto_Binarize.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # AUTO-GENERATED by tools/checkspecs.py - DO NOT EDIT
 from __future__ import unicode_literals
-from ..model import Binarize
+from ..model_bkp import Binarize
```
Comment (effigies) [inline nipype/interfaces/freesurfer/tests/test_auto_Binarize.py]: Something seems to have gone wrong. I'm guessing `model_bkp.py` is a backup file you have in your local repository? That's interfering with the proper behavior of `make specs`. Could you remove it, re-run `make specs` and add the updated auto tests?

Comment (achetverikov) [reply nipype/interfaces/freesurfer/tests/test_auto_Binarize.py]: Ah, right. I forgot that I made a backup and was looking at the error message with no clue to what's going on. I did as you suggested, hope that will work.

File: nipype/interfaces/freesurfer/model.py [hunk 2]
```diff
@@ -1144,7 +1144,7 @@ class Label2VolInputSpec(FSTraitedSpec):
     invert_mtx = traits.Bool(
         argstr='--invertmtx', desc='Invert the registration matrix')
     fill_thresh = traits.Range(
-        0., 1., argstr='--fillthresh %.f', desc='thresh : between 0 and 1')
+        0., 1., argstr='--fillthresh %f', desc='thresh : between 0 and 1')
```
Comment (effigies) [inline nipype/interfaces/freesurfer/model.py]: Let's go ahead and change this to `%g` (and remake auto tests).

Also, in the Label2Vol docstring, you'll need to change `--fillthresh 0` to `--fillthresh 0.5`.

**Comment by effigies** [comment]
Hi @achetverikov, do you have a couple minutes to finish this off? I think it's just about done.

**Comment by achetverikov** [comment]
right, I added the changes you suggested merged with master again, ran make specs, and so on 
I hope I didn't do anything that shouldn't have been done

**Pull request merged**